contentcuration/contentcuration/static/js/edit_channel/router.js

```diff
@@ -22,7 +22,6 @@ var ChannelEditRouter  = Backbone.Router.extend({
 		var ChannelManageView = require("edit_channel/new_channel/views");
 		var channel_manager_view = new ChannelManageView.ChannelListPage ({
 			el: $("#channel-container"),
-			collection: this.channelCollection
```

Comment:
I assume this is removed because it's just not being used for anything or it is set later?

Reply:
Yes, it is not used in the constructor function, nor is the collection property of this view used anywhere in the view.